### ISSUE TITLE
Training with new dataset from previous .dat file

### ISSUE THREAD

**Issue by xhuvom** [davisking/dlib#1590]
I am following the example "dnn_metric_learning_on_images_ex.cpp" to train the face recognition model with my additional dataset. I see when it runs, it creates / resumes from a sync file in the line:

` trainer.set_synchronization_file("face_metric_sync", std::chrono::minutes(5));`
and save the training output to 
` serialize("metric_network_renset.dat") << net;`

However, I am curious whether it is possible to train the face-recognition model resuming from previous pretrained model provided by dlib named `dlib_face_recognition_resnet_model_v1.dat`.
Any suggestion will be highly appreciated.

**Comment by xhuvom** [comment]
The primary curiosity for this if the accuracy could be improved further by training with the newly released [VGGFace 2](http://www.robots.ox.ac.uk/~vgg/data/vgg_face2/) dataset containing 3.3M+ faces or parametric tuning on layers.

**Comment by davisking** [comment]
Unfortunately, I didn't save the sync file.  The public model I saved was converted from one containing batch norm layers to affine, so the batch norm state was lost, which makes loading it into a model with batch norm layers annoying.
